perfzero/lib/perfzero/benchmark_method_runner.py

```diff
@@ -79,7 +83,7 @@ def _run_internal(benchmark_method, harness_info, site_package_info,
     else:
       tpu = None
     class_instance = utils.instantiate_benchmark_class(
-        benchmark_class, output_dir, config.root_data_dir, tpu=tpu)
+        benchmark_class, model_output_dir, config.root_data_dir, tpu=tpu)
```

Comment:
can we pass all these by kwargs so it's more readable and less error prone (e.g if some re-ordering of the kwargs happens later)

Reply:
Thanks for the suggestion. Good idea. Changed to passing by kwargs.